### ISSUE TITLE
spring-based animated MST properties

### ISSUE THREAD

**Issue by rbuels** [GMOD/jbrowse-components#44]
flowAnimator in the rbuels_junk branch has the beginnings of this: https://github.com/GMOD/jbrowse-components/compare/rbuels_junk?expand=1#diff-c6daf6500d7e92631a04b09422fde771

Basically I envision an MST model that has a current value, a stiffness, and a damping, and is updated with some kind of flow action that will animate the current value to the new target.

Basically an MST version of https://github.com/chenglou/react-motion

**Comment by rbuels** [comment]
here's a good thing about the runge-kutta algorithm: https://www.myphysicslab.com/explain/runge-kutta-en.html

**Comment by rbuels** [comment]
```js
const animatedNumber = types.model({
   animatedValue: types.number,
   value: types.number,

  stiffness: types.number,
  damping: types.number,
})
.actions( self => ({
    animateTo(newValue) {
       // start an animation that changes animatedValue, and then updates value when the animation finishes
   })
}))

```

**Comment by cmdcolin** [comment]
@ihh requested animated zoom :)

**Comment by rbuels** [comment]
fixed with merge of #895
